test/IECore/MeshTangentsOpTest.py

```diff
@@ -63,8 +63,8 @@ def testSimpleTriangleWithNoIndices( self ) :
 		self.assert_( "tTangent" in mesh )
 		self.assert_( mesh.arePrimitiveVariablesValid() )
 		
-		self.assertEqual( mesh["sTangent"].interpolation, IECore.PrimitiveVariable.Interpolation.FaceVarying )
-		self.assertEqual( mesh["tTangent"].interpolation, IECore.PrimitiveVariable.Interpolation.FaceVarying )	
+		self.assertEqual( mesh["sTangent"].interpolation, IECore.PrimitiveVariable.Interpolation.Vertex )
```

Comment:
can you explain why these change? it probably makes sense but i don't have the energy to think it through right now...